### PR TITLE
Allow index templates to be reseeded with new replica and primary shard values

### PR DESCRIPTION
**Pull request by ewolinetz**
Resolves inability to use workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1697728

### CODE REVIEW

**Comment by richm** [comment]
/test full-integ-aws
possible flake

File: elasticsearch/init/0100-seed-index-templates [hunk 1]
```diff
@@ -21,7 +21,13 @@ info Adding index templates
 shopt -s failglob
 for template_file in ${ES_HOME}/index_templates/*.json
 do
-    sed -i "s,\$REPLICA_SHARDS,$REPLICA_SHARDS," $template_file
+
+  if [ -f ${template_file}.bkup ]; then
```
Comment (richm) [inline elasticsearch/init/0100-seed-index-templates]: formatting is off here

**Comment by richm** [comment]
This seems to break the cluster functionality test:
```
SUCCESS after 1.505s: test/cluster/functionality.sh:130: executing 'curl_es_pod 'elasticsearch-cdm-oyb9af5r-1-68f59cc8dc-pxz2c' '/_template/com.redhat.viaq-openshift-operations.template.json' --request HEAD --head --output /dev/null --write-out '%{response_code}'' expecting success and text '200'
Running test/cluster/functionality.sh:130: executing 'curl_es_pod 'elasticsearch-cdm-oyb9af5r-1-68f59cc8dc-pxz2c' '/_template/com.redhat.viaq-openshift-operations.template.json.bkup' --request HEAD --head --output /dev/null --write-out '%{response_code}'' expecting success and text '200'...
FAILURE after 0.902s: test/cluster/functionality.sh:130: executing 'curl_es_pod 'elasticsearch-cdm-oyb9af5r-1-68f59cc8dc-pxz2c' '/_template/com.redhat.viaq-openshift-operations.template.json.bkup' --request HEAD --head --output /dev/null --write-out '%{response_code}'' expecting success and text '200': the output content test failed
Standard output from the command:
404
There was no error output from the command.
```
For each template file, it expects there to be a template in ES with the file name - but there are no templates in ES with the name with .bkup

**Comment by ewolinetz** [comment]
tests passing -- squashed, PTAL

**Comment by richm** [comment]
The errors do not appear to be flakes.  I don't see what the problem is in the elasticsearch or fluentd logs.

**Pull request merged**